### ISSUE TITLE
Spiteful Storm trinket dmg done in sim is much lower than in game

### ISSUE THREAD

**Issue by Starianor** [seriallos/raidbots-issues#393]
# If your bug is related to an incorrect tooltip, check the Full HTML Results before reporting! https://github.com/seriallos/raidbots-issues/issues/288

**Describe the bug**
I recently acquired the Spiteful Storm trinket from heroic Raszageth, and in doing some 6 minute patchwerk sims, it seemed to be simming lower than I thought it should. So I checked the detailed sim report and it consistently shows it as 0.9% of my dmg, or about 250k for a 6 minute fight. But in the game, in the few pulls I used it on mythic Terros, it was always doing at least 2% of my damage or more, and is doing 250k dmg is half the length of time. I tested on some training dummies later in town after raid and same thing. I think maybe the proc rate in the sim is lower than it should be.

**Expected behavior**
I just think the trinket should be proccing more in the sims, or the dmg it does should be higher or something.

**Link a sim with the problem (if available)**

https://www.raidbots.com/simbot/report/oNiA8AEuRzD33D52qikN8i/simc

**Character or /simc input**

For "Load from SimC Addon":

Paste your /simc input

# Starvoker - Devastation - 2023-02-07 23:09 - US/Proudmoore
# SimC Addon 10.0.5-01
# WoW 10.0.5.47967, TOC 100005
# Requires SimulationCraft 1000-01 or newer

evoker="Starvoker"
level=70
race=dracthyr
region=us
server=proudmoore
role=spell
professions=herbalism=100/inscription=92
spec=devastation

talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDkAAAAAAgiWKkIkQCJJJtkENi0CJgkIBJJSkIJJ

# Saved Loadout: Raid Build
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDkAAAAAAgiWKSiIJkQSk0SS0ISLkIhkIBJJSESSC
# Saved Loadout: Pure Single Target
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDkAAAAAAgiWKkIkQCJJJtkENi0CJgkIBJJSkIJJ
# Saved Loadout: Sennarth build
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDAAAAAAAKKhkISSkQSk0SSSLh0SkIhkAkkkEhkkA
# Saved Loadout: Broodkeeper
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDIAAAAAAooFSiIJRCSkUSSSLItQiESScASSiEhkkA
# Saved Loadout: Kurog Grimtotem
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDAAAAAAAKapIJikEJIRSLJJNi0CJSIJSQSiEhkkA
# Saved Loadout: Raszageth
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhTDCAAAAAAKahkISSkgEJlkk0g0CJiIJSQSiEhkkA
# Saved Loadout: Dathea
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDAAAAAAAKapIJikEJIRSJJJtESLkIhkwBQSSSESSC
# Saved Loadout: General Council
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDIAAAAAAoohIJSkkgkDkUSSSjItgIhkIBJJSISSC
# Saved Loadout: M+
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhTDCAAAAAAKaIJikEJkEJtkk0CSLkIhkEIJJSESSC
# Saved Loadout: Eranog
# talents=BsbBnWR3R1Z1HmbUKIpvm3XXhDkAAAAAAgiWKSiQCJkkDk0SS0ISLkIhkIBJJSESSC

head=,id=200381,bonus_id=6652/7936/7981/8095/8828/1489
neck=,id=193001,gem_id=192948/192948/192987,bonus_id=8836/8840/8902/8960/8784/8782/8802/8793/8846,crafted_stats=49/36
shoulder=,id=200383,bonus_id=8817/41/8822/7977/8826/8972/1485/8767
back=,id=144111,enchant_id=6598,bonus_id=7977/42/8822/8819/9144/8972/3301/8767
chest=,id=200378,enchant_id=6625,bonus_id=6652/8830/8974/7977/1495/8767
wrist=,id=193693,enchant_id=6580,bonus_id=7977/6652/7937/8822/8819/9144/8972/1633/8767
hands=,id=200380,bonus_id=7980/6652/8829/1485/8767
waist=,id=133805,gem_id=192948,bonus_id=8974/7977/6652/8822/8818/9144/3297/8767/8780
legs=,id=193426,enchant_id=6541,bonus_id=8836/8840/8902/8802/8846/8793,crafted_stats=36/49
feet=,id=193460,enchant_id=6607,bonus_id=8836/8840/8902/8960/8801/8846
finger1=,id=195526,enchant_id=6562,bonus_id=6652/7936/7979/1472/8767
finger2=,id=192999,enchant_id=6562,gem_id=192948,bonus_id=8836/8840/8902/8780/8802/8793/8845/8174,crafted_stats=49/40
trinket1=,id=194309,bonus_id=7980/6652/1485/8767
trinket2=,id=133641,bonus_id=7977/6652/9144/8972/3290/8767
main_hand=,id=195519,enchant_id=6643,bonus_id=6652/7980/1485/8767

### Gear from Bags
#
# Lovely Charm Bracelet (1)
# wrist=,id=49916
#
# Sharpeye Bracers (402)
# wrist=,id=109873,bonus_id=8969/7977/6652/7937/8822/8819/9144/3287/8767
#
# Galerattle Gauntlets (395)
# hands=,id=193752,bonus_id=8967/7977/42/8822/8817/9144/1617/8767
#
# Ohn'ahran Falconer's Cinch (382)
# waist=,id=192003,bonus_id=6652/7936/1485/5864/8767
#
# Legguards of the Awakened (389)
# legs=,id=200382,enchant_id=6540,bonus_id=7979/6652/8827/1472/8767
#
# Sharpeye Greaves (405)
# feet=,id=109791,bonus_id=8970/7977/6652/8822/8818/9144/3290/8767
#
# Platinum Star Band (411)
# finger1=,id=193708,enchant_id=6562,bonus_id=7977/6652/7937/9144/8972/1633/8767
#
# Conjured Chillglobe (389)
# trinket1=,id=194300,bonus_id=6652/7979/1472/8767
#
# Iceblood Deathsnare (402)
# trinket1=,id=194304,bonus_id=6652/7980/1485/8767
#
# Azure Arcanic Amplifier (392)
# trinket1=,id=198407,bonus_id=9135/6652/1446/5865/8767
#
# Ruby Whelp Shell (395)
# trinket1=,id=193757,bonus_id=8967/7977/6652/9144/1617/8767
#
# Tome of Unstable Power (398)
# trinket1=,id=193628,bonus_id=8968/7977/6652/9144/1620/8767
#
# Crackling Codex of the Isles (402)
# off_hand=,id=194879,bonus_id=8836/8840/8902/8801/8845,crafted_stats=32/40

# Checksum: 6a294caa

**Screenshots**

**Additional context**
Here is one log from a 3.5 min fight where it did 262k dmg: https://www.warcraftlogs.com/reports/Tg21BYQq3XNdyKPm/#type=damage-done&source=8&fight=30
And here is a longer fight where I died after 5:18 and it did almost 500k dmg: https://www.warcraftlogs.com/reports/Tg21BYQq3XNdyKPm/#type=damage-done&source=8&fight=31

**Comment by seriallos** [comment]
Yeah, I've passed this on to SimC devs previously. The issue seems to be pretty specific to Dev Evokers and how some of the channels aren't properly flagged to proc this trinket.
